python/sglang/srt/layers/moe/ep_moe/layer.py

```diff
@@ -4,12 +4,22 @@
 import torch
 from torch.nn import Module
 
+_enable_jit_deepgemm = False
 try:
     from deep_gemm import (
         get_col_major_tma_aligned_tensor,
+        m_grouped_gemm_fp8_fp8_bf16_nt_contiguous,
         m_grouped_gemm_fp8_fp8_bf16_nt_masked,
     )
+    from sgl_kernel import silu_and_mul
 
+    from sglang.srt.layers.quantization.fp8_kernel import (
+        sglang_per_token_group_quant_fp8,
+    )
+    from sglang.srt.utils import DeepEPMode, get_bool_env_var
+
+    if get_bool_env_var("SGL_ENABLE_JIT_DEEPGEMM", default="false"):
+        _enable_jit_deepgemm = True
```

Comment:
This part is duplicated with code: https://github.com/sgl-project/sglang/blob/b5be56944b6eb61b44866011f157e8df0e563bd7/python/sglang/srt/layers/quantization/deep_gemm.py#L29

We may import it directly.